confirmed_malicious.rego

```diff
@@ -1,27 +1,31 @@
-package policy
+# METADATA
+# title: Confirmed Malicious
+# description: |
+#   Return a violation if the pacakge or author is tied to known malicious behavior
```

Comment:
Should these be reworded like "Blocks packages or authors tied to known malicious behavior"?

Reply:
Updated